Gameplay/entities/player/PlayerCamera.cpp

```diff
@@ -10,28 +10,23 @@
 
 Hachiko::Scripting::PlayerCamera::PlayerCamera(GameObject* game_object)
 	: Script(game_object, "PlayerCamera")
-	, _relative_position_to_player(math::float3::zero)
-	, _player(nullptr)
-	, _follow_delay(0.0f)
+	, _relative_position_to_player(math::float3(0.0f, 19.0f, 13.0f))
+	, _objective(nullptr)
+	, _follow_delay(0.6f)
 {
```

Comment:
should we create serialized variables instead of magic numbers so we can change the camera distance and stuff form engine?